backend/dataall/core/environment/api/resolvers.py

```diff
@@ -360,6 +361,17 @@ def get_parent_organization(context: Context, source, **kwargs):
     return org
 
 
```

Comment:
One note out of the scope of this PR: `core` is not following the standard we have set for the `modules`, there is no a clear separation between the `api`, `services` and `db` layer. Balint refactored `organizations` and I refactored `vpc` we still need to refactor `environments`, `groups`, `permissions`, `stacks`